corehq/ex-submodules/casexml/apps/phone/utils.py

```diff
@@ -7,3 +7,11 @@ def get_restore_response_class(domain):
     if BLOBDB_RESTORE.enabled(domain):
         return BlobRestoreResponse
     return FileRestoreResponse
+
+
+def delete_sync_logs(before_date, limit=1000):
```

Comment:
Is the plan to put this in a celery task? Should this not be called recursively (or do an iteration) to ensure that all logs prior to `date` are removed?